Resources/Prototypes/_CP14/Recipes/Construction/Graphs/Walls/WallBrick.yml

```diff
@@ -0,0 +1,16 @@
+#- type: constructionGraph
```

Comment:
don't add non-working prototypes even with comments

Reply:
You can close this PR.
I want make new PR with this update and fixed prototypes.

Reply:
ok, if you want